src/components/input/MultiSelect.vue

```diff
@@ -75,7 +75,14 @@
 
     <!-- Custom option row: square checkbox + label (unchanged layout/colors) -->
     <template #option="slotProps">
-      <div class="flex items-center gap-2">
+      <div
+        class="flex items-center gap-2"
+        :style="
+          popoverMinWidth || popoverMaxWidth
+            ? `${popoverMinWidth ? `min-width: ${popoverMinWidth} !important;` : ''} ${popoverMaxWidth ? `max-width: ${popoverMaxWidth} !important;` : ''}`
```

Comment:
Do we need the `!important`s?
Also, could this use `cn()` and maybe compute the value in the script?